### ISSUE TITLE
Clipboard and File command names inconsistent

### ISSUE THREAD

**Issue by mjross** [Komodo/KomodoEdit#2675]
### Summary

In the Edit > Preferences > Key Bindings dialog, almost all of the commands have the form "[category]: [command within that category]", e.g., "Editor: Zoom Font Size Down". But the three Clipboard commands and the four File commands are named differently, with hyphens instead of colons:
- Clipboard - Copy View File Name
- Clipboard - Copy View Full Path
- Clipboard - Copy View Directory Path
- File - Copy
- File - Delete
- File - Move
- File - Rename

Initially, I suspected the difference might be a way to indicate that those commands are not available from the menus, but then I realized that there are dozens if not hundreds of commands listed in that dialog box that do not appear to be directly accessible from any of the menus, and yet all of those commands are named using the standard format.

This is admittedly not a critical issue, but a quick search revealed that this problem does not appear to have been reported earlier, and perhaps it could be a quick fix.

### Platform Information
Komodo IDE 10.2.2 on Windows 10 Pro

**Comment by mitchell-as** [comment]
Thanks for the hawk-eyed report!

**Comment by mjross** [comment]
Certainly. I wasn't sure if there was a legitimate reason for the hyphens instead of colons.
